app/src/main/java/com/sci/recipeandroid/feature/onboarding/ui/adapter/OnBoardAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package com.sci.recipeandroid.feature.onboarding.ui.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.sci.recipeandroid.databinding.ItemViewOnboardBinding
+import com.sci.recipeandroid.feature.onboarding.ui.model.OnBoardModel
+import com.sci.recipeandroid.feature.personalize.ui.viewholder.OnBoardViewHolder
+
+class OnBoardAdapter: RecyclerView.Adapter<OnBoardViewHolder>() {
+    private var List = emptyList<OnBoardModel>()
```

Comment:
"list" not "List"

Reply:
Update in e2ceaf1519d2ad526bf9d419752063a820c4ad13